pkg/querier/http.go

```diff
@@ -218,6 +218,18 @@ func (q *Querier) TailHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	tenantID, err := tenant.TenantID(r.Context())
+	if err != nil {
+		level.Error(logger).Log("msg", "error getting tenant id", "err", err)
+		serverutil.WriteError(httpgrpc.Errorf(http.StatusBadRequest, err.Error()), w)
+		return
+	}
+	level.Info(logger).Log("msg", "starting to tail logs", "tenant-id", tenantID, "selectors", req.Query)
```

Comment:
Suggestion for consistency

```suggestion
	level.Info(logger).Log("msg", "starting to tail logs", "tenant", tenantID, "selectors", req.Query)
```